calicoctl/commands/node/diags.go

```diff
@@ -22,10 +22,12 @@ import (
 	"path/filepath"
 	"strings"
 	"time"
+	"bufio"
+	"bytes"
 
 	"github.com/docopt/docopt-go"
 	log "github.com/sirupsen/logrus"
-	shutil "github.com/termie/go-shutil"
```

Comment:
nit: I much rather explicitly name the import when import name is different than the package name (not a good practice) like in this case package is `shutil` (and import is `go-shutil`) and we use `shutil` in the code, so I like to make it explicit.